feeds/wlan-ap/wlan-ap-config/files/etc/uci-defaults/99-tip-network

```diff
@@ -6,4 +6,18 @@ uci set network.wan.metric=1
 uci set network.lan.metric=10
 uci set network.wan.vlan_filtering=1
 uci set network.lan.vlan_filtering=1
+
+# Only add firewall rule if it doesn't already exist
+if ! uci show firewall | grep -q "Lan SSH Rule"
+then
+	uci add firewall rule
+	uci set firewall.@rule[-1].src='lan'
```

Comment:
Note B:

Line 13 becomes:   uci set firewall.rule='lan_ssh_rule'
Line 14 and on become:uci set firewall.lan_ssh_rule.<option>=<value>

Reply:
Minor change: Line 13 would be uci set firewall.lan_ssh_rule=rule